docs/BreakingChanges/149 Workflow now throws original exception instead of NullReferenceException in some cases.md

```diff
@@ -0,0 +1,34 @@
+## 149: Workflow now throws original exception instead of NullReferenceException in some cases
+
+### Scope
+Minor
+
+### Version Introduced
+4.7
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+In the .NET Framework 4.6.2 and earlier versions, when the Execute method of a workflow activity 
+throws an exception with a `null` value for the `P:System.Exception.Message` property, the 
+System.Activities Workflow runtime throws a `NullReferenceException`, masking the original 
+exception.
+
+In the .NET Framework 4.7, the previously masked exception is thrown. 
+
+- [ ] Quirked 
+- [ ] Build-time break 
+
+### Recommended Action
+If your code relies on handling the `NullReferenceException`, change it to catch the exceptions 
+that could be thrown from your custom activities.
+
+### Affected APIs
+* `M:System.Activities.CodeActivity.Execute`
```

Comment:
* `M:System.Activities.CodeActivity.Execute(System.Activities.CodeActivityContext)`
* This one is ambiguous... maybe
   * [`M:System.Activities.AsyncCodeActivity.BeginExecute(System.Activities.AsyncCodeActivityContext,System.AsyncCallback,System.Object)`](https://apisof.net/catalog/System.Activities.AsyncCodeActivity.BeginExecute(AsyncCodeActivityContext,AsyncCallback,Object)) 
   * [`M:System.Activities.AsyncCodeActivity``1.BeginExecute(System.Activities.AsyncCodeActivityContext,System.AsyncCallback,System.Object)`](https://apisof.net/catalog/System.Activities.AsyncCodeActivity%3CTResult%3E.BeginExecute(AsyncCodeActivityContext,AsyncCallback,Object))
* There are 7 overloads of the Invoke method:
   * https://apisof.net/catalog/System.Activities.WorkflowInvoker.Invoke()